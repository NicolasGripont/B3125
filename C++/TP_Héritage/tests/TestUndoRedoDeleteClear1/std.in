UNDO
REDO
S seg1 1 1 7 5
LIST
R rect1 2 1 11 5
LIST
UNDO
LIST
UNDO
LIST
UNDO
LIST
REDO
LIST
REDO
LIST
OR or1 seg1 rect1
PC pc1 0 0 4 8 8 -1
OI oi2 or1 pc1
LIST
CLEAR
LIST
UNDO
LIST
UNDO
LIST
REDO
LIST
DELETE oi2
LIST
REDO
LIST
DELETE or1
LIST
UNDO
LIST
UNDO
LIST
EXIT