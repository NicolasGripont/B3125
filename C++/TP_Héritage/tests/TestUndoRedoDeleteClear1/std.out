OK
OK
OK
S seg1 1 1 7 5
OK
R rect1 2 1 11 5
S seg1 1 1 7 5
OK
S seg1 1 1 7 5
OK
OK
OK
S seg1 1 1 7 5
OK
R rect1 2 1 11 5
S seg1 1 1 7 5
OK
OK
OK
PC pc1 0 0 4 8 8 -1
OI oi2 2
	OR or1 2
		S seg1 1 1 7 5
		R rect1 2 1 11 5
	PC pc1 0 0 4 8 8 -1
R rect1 2 1 11 5
OR or1 2
	S seg1 1 1 7 5
	R rect1 2 1 11 5
S seg1 1 1 7 5
OK
OK
S seg1 1 1 7 5
OR or1 2
	S seg1 1 1 7 5
	R rect1 2 1 11 5
R rect1 2 1 11 5
OI oi2 2
	OR or1 2
		S seg1 1 1 7 5
		R rect1 2 1 11 5
	PC pc1 0 0 4 8 8 -1
PC pc1 0 0 4 8 8 -1
OK
S seg1 1 1 7 5
OR or1 2
	S seg1 1 1 7 5
	R rect1 2 1 11 5
R rect1 2 1 11 5
PC pc1 0 0 4 8 8 -1
OK
S seg1 1 1 7 5
OR or1 2
	S seg1 1 1 7 5
	R rect1 2 1 11 5
OI oi2 2
	OR or1 2
		S seg1 1 1 7 5
		R rect1 2 1 11 5
	PC pc1 0 0 4 8 8 -1
R rect1 2 1 11 5
PC pc1 0 0 4 8 8 -1
OK
S seg1 1 1 7 5
OR or1 2
	S seg1 1 1 7 5
	R rect1 2 1 11 5
R rect1 2 1 11 5
PC pc1 0 0 4 8 8 -1
OK
S seg1 1 1 7 5
OR or1 2
	S seg1 1 1 7 5
	R rect1 2 1 11 5
R rect1 2 1 11 5
PC pc1 0 0 4 8 8 -1
OK
S seg1 1 1 7 5
R rect1 2 1 11 5
PC pc1 0 0 4 8 8 -1
OK
OR or1 2
	S seg1 1 1 7 5
	R rect1 2 1 11 5
S seg1 1 1 7 5
R rect1 2 1 11 5
PC pc1 0 0 4 8 8 -1
OK
OR or1 2
	S seg1 1 1 7 5
	R rect1 2 1 11 5
S seg1 1 1 7 5
OI oi2 2
	OR or1 2
		S seg1 1 1 7 5
		R rect1 2 1 11 5
	PC pc1 0 0 4 8 8 -1
R rect1 2 1 11 5
PC pc1 0 0 4 8 8 -1